### DISCUSSION TITLE
Daily Working Log

### DISCUSSION THREAD

**Discussion by ryanpseely** [ecn310/course-project-nepobabies#4] [General]
#### Notes from Research Hypothesis Workshop (to be edited)
- Cut down on the words used in the tentative hypothesis and make it more clear what we think the end result will be due to the first incident (recessions) and less of the theory can be cut out.
- Variables are good for the hypothesis test.
- The mechanism portion of the discussion post for testing the hypothesis can be cut down by removing the moderator variables because while they will add to the analysis, they are moderator variables and not the main thing the hypothesis is looking at.
- Clarify what kind of statistical analysis we will be examining. As of now I am thinking of using difference of means/one-way ANOVA.

Ryan and Rachel

**Comment by ryanpseely** [comment]
### October 24, 2023
**Discussion with Professor Buzard regarding research hypothesis workshop**

#### Editing the theory: 
- Using ideas from the literature, we can come up with our own theory as long as it is based in economic methodologies/ways of thinking. That is, we do not have to derive the theory explicitly from the literature. 
- We should include in the theory about how labor market entrants have their incentives altered during recessions and may be more incentivized to rely on their connections in order to optimize their chances of getting a job. 
#### Editing the hypothesis: 
- The theory and hypothesis should also be more focused on the supply side of labor as opposed to the firm/demand-side. 
#### Data examination: 
- Look for data in GSS on when someone got their job and the other variables like major course of study and parent industry.
- Look in GSS data to see if people are more likely to have the same majors as their parent's industry during recessions.
- Must see how specific the data is on how long people have had their jobs.
- Using a variable for labor market saturation, probably unemployment rate, may be better and easier than just looking at the dates of recessions.

Ryan

**Comment by kbuzard** [comment]
@rpseely There is a nice little section in the Daniels & Minot book on cross tabulations. I'm not sure when we'll get to it, but I think it's what you need to answer the question you were asking about what years your different college degree variables show up. It's Section 6.6, starting on page 91.
- To get around the "too many values" problem, here is what I would suggest:
   1. Create a new variable that is just a count of the non-missing values (for these purposes, you don't care how many people are in each major for each year; just how many people HAVE a major). Here is generic code you can customize: `egen non_missing_count = total(!missing(myvar))`
   2. Then, do the cross tab with year, something like `tab year non_missing_count, col`

**Comment by rhrabino** [comment]
**October 31, 2023**

- Today Ryan and I worked on making a do file in Stata exploring our data. We had a little trouble uploading the do file to our Wiki master documentation because we couldn't figure out how to pull it from the GitHub desktop. Right now, we copied and pasted our do-file into an issue.
- We created an issue for data exploration and analysis
- We also updated our Wiki master documentation. We added links to the daily working log, a new issue for data, causal diagrams, an updated link to the lit review, and a new issue for task documentation. 
- We created a task documentation issue

**Comment by ryanpseely** [comment]
#### November 1, 2023
- I updated the task documentation to reflect the steps we must take that were suggested by Prof. Buzard in my meeting with her.
- I updated the hypothesis and theory as recommended by Prof. Buzard.

**Ryan**

**Comment by ryanpseely** [comment]
#### November 6, 2023
- Closed the log for the problem we had [uploading the initial data exploration do-file](https://github.com/ecn310/course-project-nepobabies/issues/5) as we have solved that issue.
- Added a description to [this issue](https://github.com/ecn310/course-project-nepobabies/issues/12) about using the unemployment rate variable in our data analysis. I also renamed the issue for better readability.
- I added some description to [this issue](https://github.com/ecn310/course-project-nepobabies/issues/11) of needing to determine how the GSS measures the time a respondent has held a job and changed its name for better readability.
- I added description to [this issue](https://github.com/ecn310/course-project-nepobabies/issues/10) regarding the new variable we must make for matching college major to parent's industry and renamed it for readability.

**Ryan**

**Comment by ryanpseely** [comment]
### November 6, 2023
Notes from meeting with Professor @kbuzard during office hours:
- It may be best to isolate a single year and and examine how respondent's industries match up with their parent's industries, examine how this changes over the course of the year, and then see how this interacts with the unemployment rate.
- We must find the date function in Stata to see how it is measured so that we can go back and see how long people have had their jobs, in order to ultimately make a variable on whether or not people got their job during a recession.
- We must make a decision rule for how we put when people got into a job into bins.
- We are also planning to meet with @eldreddyl on Wednesday to start creating these new variables. 

Ryan and @rhrabino

**Comment by ryanpseely** [comment]
### November 8, 2023
Today, we met with @eldreddyl to discuss how we can create new variables to move our project forward.
- First we broke down what variables we had, what variables we needed, and how it responds to our research question.
- Then we talked about how we could add in the unemployment rate data, but ultimately we probably will not go down this route because...
- Using the command `browse year dateintv indus10 paind10` we found that in every year in which the GSS was conducted from 1975-2022, the date a respondent was interviewed is given. This solves a major issue we had which was the need to create a new variable for respondents being interviewed during a recession or not so that we could do a hypothesis test on different labor market patterns for nepobabies.
- We then refined our hypothesis (not the actual research hypothesis, just for the hypothesis test) to be the following:
P (same job as 1+ parent) in bad times > P (same job as 1+ parent) in good times
- We are now going to, in an excel sheet, create a dummy variable for recessions.
- We also now have to create a dummy variable for being a nepobaby, i.e. being in the same industry as a parent.

The following photos are some guiding notes that Dylan wrote on the whiteboard:

<img src="https://github.com/ecn310/course-project-nepobabies/assets/143651368/b045a466-8df1-416e-b04f-7bcde956c6a0" width=33% height=33%>

<img src="https://github.com/ecn310/course-project-nepobabies/assets/143651368/bd959fa1-af65-43fe-a7a4-ef3492e06400" width=33% height=33%>

<img src="https://github.com/ecn310/course-project-nepobabies/assets/143651368/0b2d8e9c-2689-4cd1-bd34-c138f195f7f2" width=33% height=33%>

**Ryan and Rachel**

**Comment by rhrabino** [comment]
### November 8, 2023
Ryan and I made an excel file titled "recessiondates" which lists all the months from 1975-2021 and whether or not they fall under a recession. The excel file was uploaded to our onedrive folder.

**Comment by ryanpseely** [comment]
### November 10, 2023
Today we worked on updating our dataset to get it into a good place for when we start performing our hypothesis test(s).
- We made the excel sheet called `recessiondates.dta` into a Stata dataset so that we can merge it with the GSS data in Stata. We added this into OneDrive.
- We cleaned the GSS dataset so that we have only the variables we need for our hypothesis test and some other interesting variables that might be moderators. We added this new dataset into OneDrive and it is called `GSSclean.noexcel.dta` and the "noexcel" refers to the fact that it is still missing the recession variable that will have to be merged from the excel sheet we turned into a Stata dataset today.

**IMPORTANT GOING FORWARD:** We still need to merge these datasets!

**Ryan and Rachel**

**Comment by ryanpseely** [comment]
### November 14, 2023
- I added the `GSSclean.noexcel.nogender.dta` dataset containing our cleaned data to the code of our repository. It still needs to be replaced as it is currently missing the variable for respondent age and respondent gender.
- I also added the `recessiondates.dta` dataset to the code of our repository, and it contains part of the data we need to create a dummy variable for whether or not a respondent was interviewed during a recession.
- I found an article that could be helpful for our lit review and might give us something in the theory that we did not have. However, I ran into a line in the introduction of the article that might actually say the opposite of what we are looking for. In that case, should we still include it? Or just say that maybe it does not apply for what we are researching. @kbuzard If we have time in class tomorrow, maybe I can show it to you.
- Professor Buzard and I discussed project needs in the [exercise repo](https://github.com/ecn310/exercises/discussions/14) and we discussed how we are going to go forward with our project in terms of cleaning the data, documenting our work, and creating new variables to complete our hypothesis tests.

_I meant to upload this last night but forgot to, but this was all completed on November 14_

**Comment by ryanpseely** [comment]
### November 15, 2023
- In class, @kbuzard said that we will most likely use a two-sample t test to compare the means between the percent of young adults who are in the same industry as their parents during recessions and not during recessions.
- @eldreddyl helped us update and upload our do-file with the correct variables for data cleaning.

**Comment by ryanpseely** [comment]
### November 15, 2023
- After class, I went to office hours to discuss the article I found with Professor Buzard. The result of that meeting is that we will use that article in our lit review even though it does not necessarily support our work. I will expound upon this in the [lit review discussion.](https://github.com/ecn310/course-project-nepobabies/discussions/3)

**Comment by ryanpseely** [comment]
### November 16, 2023
- I closed the lit review issue to reduce redundancy.
- I documented the updates we made to our do-file that will eventually update the cleaned GSS dataset.

**Comment by ryanpseely** [comment]
### November 17, 2023
- I updated the lit review discussion to include some boxes we can check off as we complete tasks. I also thought of going from the opposite direction for a new paper supporting our research, seeing how parents will help their children find jobs at their firm/in their industry.
- Created `nepobaby` dummy variable for a respondent having the same industry as their mother or father. An issue is that for when a respondent did not mark what their industry was, or what the GSS labels `iap`, and the respondent did not mark their parent's industry, we get false matches for being a nepobaby.
- We started to paste the `year` and `dateintv` variables together so that we can eventually label interview dates, and therefore respondents, as being in or not in recessions.

**Reply by eldreddyl** [reply]
https://www.stata.com/manuals/m-5substr.pdf

**Reply by eldreddyl** [reply]
A variable that is the full date of when the person was interviewed. Something like this

|  year | dateintv  | fullyear  |
|---|---|---|
| 1975  | 314  |  31475 |
|  2001 | 109  | 10901  |

**Comment by ryanpseely** [comment]
### November 20, 2023
- I created an issue for [Overleaf and Project Write-Up](https://github.com/ecn310/course-project-nepobabies/issues/14) and asked Prof. Buzard about the template we should be using for the final report.
- I closed an issue related to the literature review, as anything related to that is in a [discussion](https://github.com/ecn310/course-project-nepobabies/discussions/3).
- I updated an issue about a specific problem with the hypothesis and re-titled [the issue](https://github.com/ecn310/course-project-nepobabies/issues/8) to be broader and so that we can keep working on hypothesis test and the more theoretical hypothesis, as well as the accompanying causal diagram.

**Comment by ryanpseely** [comment]
### November 21, 2023
Professor Buzard performed a documentation check today, so @rhrabino and I completed the following tasks to update our documentation and wiki in response to her comments...
- We renamed the [nepobaby variable issue](https://github.com/ecn310/course-project-nepobabies/issues/10) to reflect how our hypothesis check has changed direction towards measuring nepotism.
- We closed the issue for [GSS specificity of job length](https://github.com/ecn310/course-project-nepobabies/issues/11) because we are no longer using that to test our hypothesis.
- We closed the [unemployment rate variable](https://github.com/ecn310/course-project-nepobabies/issues/12) issue because we no longer need that for our hypothesis check
- We also updated the wiki to show the links to our data sets and include the Overleaf and project write-up
- We made an [issue for creating the reccession dates variables](https://github.com/ecn310/course-project-nepobabies/issues/15) to track our progress on this.
- We began updating/reorganizing the wiki/master documentation.

**Reply by kbuzard** [reply]
@ecn310/nepobabies 

Here's the place I thought the task documentation needed to be updated:

> Link to Task Documentation
> `https://github.com/ecn310/course-project-nepobabies/issues/7`

It's at the very bottom, but it's a closed issue.

**Reply by ryanpseely** [reply]
@kbuzard I replaced that link and largely reorganized the wiki for better use and readability.

**Comment by ryanpseely** [comment]
### November 22, 2023
- Added context to the hypothesis (test) and causal diagram [issue](https://github.com/ecn310/course-project-nepobabies/issues/8).
- I looked through our income variable and found a problem with the variable `rincome` we were intending to use. So I found two other variables `realinc` and `coninc` that both appear accurate and span all the survey years. We must now choose which one we plan to use, and to do so we will probably visit office hours to understand which one is better to use. We also need assistance with finding the right codebook from the [Stata documentation website](https://gss.norc.org/Get-Documentation).
- I updated the [wiki](https://github.com/ecn310/course-project-nepobabies/wiki) and reformatted for better readability and organization.

**Comment by ryanpseely** [comment]
### November 23, 2023
- Professor Buzard pointed out an issue with how we planned to categorize nepobabies as recession and non-recession for our planned difference of means test. By not taking into account when respondents obtained their jobs, we could be incorrectly categorizing the recession and non-recession groups, leading to noisy data at best and results that are the exact opposite of what we intended to measure at worst.
- Therefore, we will need to take into account how long people have had their jobs. We saved one variable called `yearsjob` in our cleaned dataset, which does not measure exact employment length, but could nonetheless be useful. We would have to do some additional work to categorize respondents as recession or non-recession, but essentially we could use the `yearsjob` and `dateintv` variables to measure if people were interviewed during a recession or not and then use that information to make the categories.
- As a result of this discussion and thought, I reopened the [GSS employment length data](https://github.com/ecn310/course-project-nepobabies/issues/11) so that we can continue working on how we will match respondents to recession or non-recession labels.
- I updated the [hypothesis (test) and causal diagram issue](https://github.com/ecn310/course-project-nepobabies/issues/8) to adjust for this update to our hypothesis test.
- I updated the [wiki](https://github.com/ecn310/course-project-nepobabies/wiki) to display the reopened [GSS employment length data](https://github.com/ecn310/course-project-nepobabies/issues/11) issue.


_Below I have put together a centralized list of tasks from various issues that we hope to accomplish next week, likely with Professor Buzard during her Monday office hours_
#### 11/27 Office Hours
- [x] Discuss the value of unemployment rate vs. recession dates and noise.
- [x] Search for other employment length variables in the original GSS dataset.
- [ ] `coninc` versus `realinc`
- [ ] `iap` meaning and [GSS codebook](https://gss.norc.org/Get-Documentation) (which do we use?)
- [ ] Correcting the classification of the `nepobaby` variable due to ostensibly false matches from `iap`
- [ ] Is `unemploymentrate.monthly.xslx` formatted correctly for merging datasets?

**Comment by ryanpseely** [comment]
### November 26, 2023
- Using [ChatGPT](https://chat.openai.com/share/a1e89051-d5dd-4c2e-b824-16f1b93fb32c) I figured out how to code the variable that strings together `dateintv` and `year` to create a variable called `flintdate` (short for full interview date) that gives the month, day, and year for which respondents were interviewed.
- I updated the [recession dates issue](https://github.com/ecn310/course-project-nepobabies/issues/15) with more detail to reflect this update.
- The dataset and do-file as saved on GitHub have not been altered to reflect this change yet.
- Separately, I finished manually updating the `recessiondates.xlsx` to contain the full date. It still needs to be converted into a `.dta` file and replaced on the OneDrive and in GitHub. Additionally, we may need to create another excel sheet and manually input the monthly unemployment rate so that we can complete a hypothesis test based on that statistic as opposed to recession dates.

**Comment by ryanpseely** [comment]
### November 27, 2023
- I started working on a dataset for unemployment rates by month for all the survey years we have data for. It can be found in our [OneDrive](https://sumailsyr-my.sharepoint.com/personal/rhrabino_syr_edu/_layouts/15/onedrive.aspx?ct=1696616219053&or=OWA%2DNT&cid=87642c6f%2D751b%2Dbfc3%2D8c20%2D489c586e0930&fromShare=true&ga=1&id=%2Fpersonal%2Frhrabino%5Fsyr%5Fedu%2FDocuments%2FECN%20310%20Project)

**Comment by ryanpseely** [comment]
- monthhire = dateintv - yearsjob
- gen rec-hire = 0 for everyone
- replace rec-hire = 1 if 110173 < monthhire < 33175
- all dates must be transformed into stata dates
- see if coninc and realinc have a common ratio
- robustness check with the 2022 and without the 2022 data.
- gen nepobaby = (indus10 == paind10 & indus10 != iap) | (indus10 == maind10 indus10 != iap)
- check the previous line of code for the four cases of matching with and without iap

Check the stata dates first before data entry for unemployment rates and recession dating coding.

**Comment by ryanpseely** [comment]
### November 27, 2023
- We met with Professor Buzard to discuss the validity and tentative plan of our hypothesis test(s).
- She recommends that we go back and find the month that respondents were hired in, and then use the unemployment rate instead of recession date as our main method of testing whether or not nepotism changes throughout recessions. We can still do a test with the recession dates, but it would produce less noise and likely be a better test of what we are looking to examine to backdate respondents' hire date/month and then see how that interacts with the unemployment rate, which would again produce less noise. More on this in the [hypo. test issue](https://github.com/ecn310/course-project-nepobabies/issues/8).
- Because the month of hire is not exact, we will perform a [sensitivity analysis](https://github.com/ecn310/course-project-nepobabies/issues/11).
- We will determine our [income variable](https://github.com/ecn310/course-project-nepobabies/issues/13) after looking to see if there is a common ratio between the two ones we currently have.

**Comment by rhrabino** [comment]
### November 29, 2023

- Today we made a plan to split up the final project/report with due dates to keep us on track.
- We also made a (very) rough outline of steps to test our hypothesis.

**Reply by eldreddyl** [reply]
@ecn310/nepobabies 

Ryan mentioned at the end of class that you wanted to meet and talk through coding for your project. I'm available during the following times

### Friday, December 1st
- I have a meeting from 10-11am. Other than that, I can meet anytime before class

### Monday, December 4th
- 9:30-11:30am

**Reply by ryanpseely** [reply]
@eldreddyl This sounds great! How is 1 p.m. on Friday?

**Reply by eldreddyl** [reply]
@rpseely Perfect. That works with me. I'll meet you two in our classroom then

**Comment by rhrabino** [comment]
### December 1, 2023

- We created a new variable called `yearsjob_int` to convert these dates into integers
- We created a new variable called `yearhire` which is `year - yearsjob_int`
- We then created the variable `flintdate` to see the full date in which someone was interviewed
- To make our data cleaner, we dropped years if it was before 1976 
- We created a new variable called `flhiredate` to show the full date in which someone was hired 
- As shown in the do-file, we also tried some other commands, but they didn't work (we are working on this) 
- We are currently trying to get the full interview date and full hire date into a date format that Stata can read

**Reply by kbuzard** [reply]
> To make our data cleaner, we dropped years if it was before 1976

You'll need to be more specific when you explain this in the data section of your paper. What was is about the earlier years that led you to drop them?

**Comment by kbuzard** [comment]
@ecn310/nepobabies Everything looks good with your documentation except that I don't see anything about your plan. Are you just going to keep updating the version on the exercises repo? If so, that's fine, but you need a link to it in your wiki. You can also copy it over to your repo somewhere that makes sense for you; just make sure there's a link on the wiki so I know where to find it.
- You should add a task for documentation plan next Wednesday, and for submitting the draft of the analysis section by 8am Dec. 12 at the latest (the two new assignments I told you about in class yesterday)
- There are a lot of outstanding tasks on that plan; and some of them haven't yet been converted into task issues in this repo
  - there doesn't need to be a separate issue for each entry on the plan; there are ways to group them together that make sense and prevent an overwhelming number of open issues. You've already done this to some extent; I just want to make sure everything has a documentation home
  - In particular, you have the presentation coming up in less than a week. What is the plan for getting it pulled together? What kind of file will you use? Where do it live?

**Comment by ryanpseely** [comment]
### December 3, 2023
- Using ChatGPT extensively, I was able to code a new variable called `ymintdate` which is derived from year month interview date. This is because I used the month and year that each respondent was interviewed in to create a Stata date, i.e. that Jan. 1960 = 1, and every month after that is coded as the number of months since then - 1.
- I also created the variable `ymhiredate` which is simply `ymintdate - (yearsjob * 12)` which puts out the month that is exactly the number of years that respondents reported for having their job.
- Finally, I coded a way to correct for false positives that occurred when we originally made the `nepobaby` variable.
- I updated the code to reflect all of this.

**Comment by rhrabino** [comment]
### December 5, 2023

- Today we inputted the monthly unemployment rate data. Using data from the BLS, we coded for each individual month to have the unemployment rate value. These 938 lines of code are not in the main do-file, but rather in a separate do-file titled `urate_input.do`
- We dropped the respondents who had an age or age of hire greater than 29.
- We looked at the connection between higher unemployment (marked by the 75th percentile) and nepobabies. This did not work and showed as more nepobabies within lower unemployment.
- We looked at the connection between higher unemployment (marked by the median) and nepobabies. This worked and showed more nepobabies when there was higher unemployment! We used a two-sample t-test and a bar graph to show these results.
- We started doing a sensitivity analysis where we add and subtract months to the respondents to see if there is any change when we do that. We are working on the code but have not been able to complete it as of yet.

**Comment by ryanpseely** [comment]
### December 6, 2023
- In class tried running the do-file for `urate_input` as a `do` function in the main do-file but it did not work. When I visited Professor Buzard's office hours, she was able to get it to work.

Here are the following recommendations made by Professor Buzard on how we can improve our organization and analysis:

- [x] Put the line of code that creates the unemployment rate variable into the `urate_input.do` file. `gen unemployrate == .`
- [x] Change the line of code that drops observations based on age and age hire. We can drop people based on `agehire` alone, because if someone is younger than 30, they have of course been hired before 30.
- [x] Do a visual check using the `browse` command to examine whether dropping based on age of hire worked.
- [ ] Add a comment on why dropped based on age hire.
- [ ] Add confidence intervals to the bar graphs.
- [x] Stick to observations including and prior to 2018. BOO COVID!
- [x] Change the value for median unemployment rate. The 4.9 median we are using now is weighted down by our sample, our median should be based off of the median or average of the unemployment rate of all the months, not all the observations.

**Reply by ryanpseely** [reply]
I tried adding 95% conf. intervals to the graphs but was unsuccessful. Will keep trying!

**Comment by ryanpseely** [comment]
### December 7, 2023
I updated the main do-file for:
- organization
- dropping based on `agehire`
- changing the median unemployment rate to 5.9, the average monthly unemployment rate from 1987-2018

- I updated the `urate_input.do` file by the line of code that creates the unemployment rate variable, `unemployrate`

**Comment by ryanpseely** [comment]
### December 7, 2023
- We are largely finished with the data analysis! SO EXCITING! 
- The biggest change we made to our hypothesis test is how we define low and high unemployment. During office hours, Professor Buzard and I discussed using the average unemployment rate that is unweighted (5.9%) as opposed to the sample-weighted average (4.9%). At these different unemployment rates, splitting the sample in half and running a t-test returned two drastically different results. Therefore, we cut out the middle 50% of responses/nepobabies to examine the top and bottom quartiles with one another at the unemployment rates >=6.625% and <=4.8%, respectively. This test allowed us to reject the null hypothesis.
- We also tested the gender dynamics of nepotism between parents and children. The results were that fathers are more likely to have the same industry as sons, and mothers are more likely to have the same industries as sons.
- I looked at the demographics of nepobabies vs. the demographics of the sample, and none of the demographics tested were significantly different at the 95% confidence interval.

**Comment by ryanpseely** [comment]
### December 8, 2023
- We finished the presentation!
- We added a nepobaby vs. sample test for job security.
- We cleaned up some of the comments in the do-file.

@kbuzard Below I have attached the PDF of our powerpoint presentation for class.
[ECN 310 - Nepobaby Class Presentation.pdf](https://github.com/ecn310/course-project-nepobabies/files/13619407/ECN.310.-.Nepobaby.Class.Presentation.pdf)

**Comment by kbuzard** [comment]
I checked--`ciplot` is the right command. If you need help troubleshooting it, let me know.

Feedback on presentation
- Can you say a little more in your theory about why people might NOT exploit their family connections if they don't need to? My logic: there must be drawbacks if so many people only go to this option when the labor market is tight.
- I LOVE the inclusion of your null hypothesis!!!
- If I understood correctly, the way you're presenting the main results is to look JUST a nepo babies, and say that if someone is a nepobaby, they are more likely to have been hired during a bad labor market. No complaints here.
  - Since different amount of people are hired during good and bad labor markets, someone might also want to know how the _proportions_ of nepobabies in the total that are hired differ between bad and good labor markets. You showed me some examples like this in office hours, didn't you?

**Comment by kbuzard** [comment]
@ecn310/nepobabies I think everything is headed in the right direction in terms of the documentation; I'm just a little worried that some of the upcoming tasks don't show up in your list of issues.
- For instance, the draft of the analysis section is due in less than 48 hours...see guidelines [here](https://github.com/ecn310/exercises/discussions/18).

**Reply by ryanpseely** [reply]
Thank you! Yes, we definitely need to and are going to start incorporating our tasks into issues more than we have been. We have started by creating this issue for the [analysis draft.](https://github.com/ecn310/course-project-nepobabies/issues/17)

**Comment by ryanpseely** [comment]
### December 10, 2023
- Rachel and I began working on our [draft of the analysis section](https://github.com/ecn310/course-project-nepobabies/issues/17) for submission before Tuesday morning. 
- We plan to submit the drafts of the data section, which we started working on today, and submitting the lit review section, which Rachel worked on last week. We are drafting the language on this [google doc.](https://docs.google.com/document/d/1_B8nqDqLiRncH2eSj00DL0Es-FJNGCcAh6oBVzfvlh0/edit)

**Comment by ryanpseely** [comment]
### December 11 & 12

#### Notes from 12/11 Office Hours
- First we asked Professor Buzard about some of the feedback she left on our presentation that suggested doing additional analysis that includes the proportion of non-nepobabies hired during recessions. After discussing this, we decided that it was important to include this analysis to demonstrate that despite there being fewer people hired in high unemployment months, there were more nepobabies hired.
- Next we talked about using the word "nepobabies" in our final report. We decided to discuss the popularization of the term in recent years, briefly, and then define it and operationalize it so that we can just use the word "nepobabies" and not have to say something like "young adults who are in the same industry as at least one parent" every time.
- We discussed how to create confidence interval plots using the `ciplot` command and how a command `ssc install ciplot` may be necessary to access it. These were added to the do-file
-  FInally we discussed whether or not the main do-file must include the original dataset given to us by Prof. Zhu. We decided to only use the `GSSclean_noRDs` file in the main do-file, and create another one showing how to get from the original GSS dataset to the one with only the variables we want and only observations that include yearsjob.


#### Notes on Analysis Draft and Submission
- Rachel and I finished and submitted the rough draft of the analysis section of our final report, along with the literature review and the data sections. We added these to the [file system](https://github.com/ecn310/course-project-nepobabies) of our repository.
- As suggested by Professor Buzard in office hours on Monday, we added a new piece of analysis to our examination of how nepotistic parent-child relationships are altered by economic conditions. We already had defined the number of nepobabies hired during high and low unemployment, so we created two new corresponding groups of non-nepobabies hired during high and low unemployment. We then took the ratios of the corresponding groups and gave the following analysis:
`We found the ratio of nepobabies hired in high unemployment to non-nepobabies hired in high unemployment, which is approximately 0.11. That same ratio, but for nepobabies and non-nepobabies hired in low unemployment, was nearly half that at 0.065. What we can extrapolate from that is that nepobabies are much more likely to rely on their parent’s for a job during highly competitive labor markets than during times with low labor market competition.`
- This provides stronger evidence that the results from our original t-test show that nepobabies are more likely to be hired during high unemployment by taking advantage of the connections of their parents.

**Comment by ryanpseely** [comment]
### December 17, 2023
- Today I worked on a little bit more of the analysis as was suggested by Prof. Buzard in office hours last week, as well as some work in Overleaf formatting outputs from Stata.
- @kbuzard Last Wednesday during office hours we discussed how we could improve upon the analysis. You suggested doing a t-test not just for the top 25 and bottom 25 percent of monthly unemployment rates, but top 25 vs bottom 75 and vice versa. I did that but then it occurred to me that the difference of means for the dummy variable just tells us what percent of observations were found in each sample. Approximately 36% of nepobabies were hired in high unemployment (u >= 6.625) and approximately 39% of nepobabies were hired during medium unemployment (4.8 < u < 6.625). But this does not really tell us anything about the two different groups in relation to one another because there is nothing standardizing the samples for the 25 vs. 75 tests like there was for the 25 vs. 25 test. 
- So I (ChatGPT) decided to do a Chi-Square test with the three groups ( hired during low, medium, high unemployment) cross-tabulated with the nepobaby variable. The results indicate that it is unlikely that the differences among the groups are random and that there is probably something going on, at least giving us enough evidence to reject the null hypothesis again. I screenshotted the table and results (taboo, I know, sorry!) and attached it below. I just want to make sure that I am interpreting these results correctly because I do not have much experience with chi-squared tests.
<img width="409" alt="Screen Shot 2023-12-18 at 2 09 19 AM" src="https://github.com/ecn310/course-project-nepobabies/assets/143651368/969a5a96-157d-4154-94e2-324a64f7df22">

**Reply by kbuzard** [reply]
@rpseely This is great! And totally okay to screenshot for informal purposes--we do it all the time. It's just not okay in something formal like a paper or report.

I think this answers really clearly the question I was asking last Wednesday: is the rate of nepobabies low when things are really good relative to the rest, or is the rate of nepobabies when things are bad high relative to the rest.

You can make a nice chart of this that also includes the percent of nepobabies in each of the three groups--I calculated it, and the percentage for groups 2 and 3 are pretty close. It also makes me wonder what you'd find if you broke the middle group out into the 2nd and 3rd quartile.
- It also points out that my worry that the months are not evenly distributed in the unemployment rate groups is valid. Something this would suggest (it's probably cutting it too close to get results into the report) is to create the groups so that there are an even number of PEOPLE in each group (well, 50% in the group 2 here). It's just another way of dividing things up. You could talk about this as a next step.

**Comment by ryanpseely** [comment]
### December 18, 2023
- I edited the analysis to make four groups of unemployment hires: low (< 4.5), mid-low (4.5 < u <= 5.4), mid-high (5.4 < u < 6.7), high (> 6.7). This showed similar results to the three groups (bottom 25th percentile, 26th through 74th percentile, and top 25th percentile.
- The t-test of high vs. low unemployment hire groups still holds with the groups based on those conditions. Mid-high is also significantly different from mid--low, but is not significantly different from high.
- I also changed the way that the examine the unemployment rates. Instead of using the manual input do-file from the BLS, I decided to use (more conveniently formatted) FRED monthly unemployment rate data that goes back a little farther (with the manually inputted BLS data, we had a few observations that were not counted because they were hired before 1987). This way, it is easier to reproduce.
